OracleWebLogic/samples/1221-multihost/create-container.sh

```diff
@@ -7,7 +7,8 @@
 . ./setenv.sh
 
 uuid=$(uuidgen)
-name=$prefix-instance-$uuid
+#Define unique server name using suffix entered by user
+name=server$2
```

Comment:
Hi @mriccell. Can you briefly describe why the change on name format for the container here?